lighthouse-core/gather/gatherers/dobetterweb/stacks.js

```diff
@@ -18,12 +18,20 @@ const fs = require('fs');
 const libDetectorSource = fs.readFileSync(
   require.resolve('js-library-detector/library/libraries.js'), 'utf8');
 
+/**
+ * @typedef JSLibrary
+ * @property {string} name
+ * @property {string} version
+ * @property {string} npmPkgName
+ * @property {string} iconName
```

Comment:
should specify which of these ends up optional in the result

Reply:
what do you mean with this comment? ^^